modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/release-notes/fragments/release-notes-tabs/index.html

```diff
@@ -6,12 +6,22 @@
 	[#assign currentTabId = (currentDisplayPaths?first)! /]
 
 	[#assign currentDisplayIds = (displayPortalURLs?last?split(currentTabId)?last)! /]
+
+	[#assign paginationParameterIndex = currentDisplayIds?index_of("&start=") /]
+
+	[#if paginationParameterIndex != -1]
+		[#assign beforePaginationParameter = currentDisplayIds?substring(0, paginationParameterIndex + 7) /]
+
+		[#assign currentDisplayIdsParameterized = beforePaginationParameter + "1" /]
+	[#else]
+		[#assign currentDisplayIdsParameterized = currentDisplayIds /]
```

Comment:
could we simplify this by assigning the varibable outside of the if, and then only resetting it if it enters the condition? Also feels odd to be doing an index + 7 and then + 1 as a string, is there a way to simplify that or make it more clear?

Reply:
Done.